wps/locales/fr_FR/wps.UTF-8.properties

```
@@ -7,15 +7,19 @@ message.error.unknow_file=Fichier non trouvé
 message.error.upload.bad_filename=Nom de fichier vide ou mauvaise extension (.model3)
 message.error.upload.existing=Fichier déjà existant, effacez-le d'abord
 message.error.upload.bad_content=Contenu invalide
+message.error.upload.modify.not_same=Le fichier doit avoir le même nom que l'original
 message.ok.file_added=Fichier ajouté
 message.ok.file_deleted=Fichier effacé
+message.ok.file_modified=Fichier modifié
 
 ui.button.file.add=Ajouter un fichier model3
 
 ui.page.list.title=Liste des model3
 ui.page.list.empty_dir=Aucun model disponibles
 ui.page.upload.title=Téléversement de fichier model3
 ui.page.conf.title=Configuration des restrictions de projet
+ui.page.confirmdelete.title=Confirmer la suppresion du fichier
```

```suggestion
ui.page.confirmdelete.title=Confirmer la suppression du fichier
```

Can you check in your IDE if you can install a spellcheck English and French ? See #52 ;-)